lesson20_functions.py

```diff
@@ -0,0 +1,18 @@
+"""
+Pick your solution to one of the exercises in this module.
+Design tests for this solution and write tests using unittest library.
+# I choose to use my own exercises:
+"""
+
+
+def add(x: int | float, y: int | float) -> int | float:
```

Comment:
Nicely done, only thing I could recommend is to use type alias for easier readability, but that's completely optional:
```python
number = int | float
def add(a: number, b: number) -> number:
    ...
```